tests/check_framework/test_translation.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.core.checks.translation import check_setting_language_code
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+
+
+class TranslationCheckTests(SimpleTestCase):
+
+    @override_settings(ROOT_URLCONF='check_framework.translation.valid_language_code_format_ll_only')
+    def test_valid_language_code_format_ll_only(self):
+        settings.LANGUAGE_CODE = "en"
```

Comment:
Does it work if you use `override_settings` for this too? That would be cleaner.

Reply:
Yes test work fine without it.